src/test/java/HomeWork/Inputs.java

```diff
@@ -0,0 +1,23 @@
+package HomeWork;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Inputs extends PreCondition {
+    @Test
+    public void inputs() {
+        driver.navigate().to("https://the-internet.herokuapp.com/inputs");
+        Assert.assertTrue(driver.findElements(By.cssSelector("input[type=number]")).get(0).getAttribute("value").equals(""));
```

Comment:
And why you use findElementS here? There's a single input in the page -> use findElement instaaed